edx_repo_tools/repo_checks/repo_checks.py

```diff
@@ -817,6 +817,22 @@ def is_relevant(self):
         return is_public(self.api, self.org_name, self.repo_name)
 
 
+@Check.register
+class MaintainersAtLargeTeam(TeamAccess):
+    """
+    Ensure that the committers-maintainers-at-large team grants Push access to every public repo in the org.
+    """
+
+    def __init__(self, api, org, repo):
+        team = "committers-maintainers-at-large"
```

Comment:
This is a mouthful but I felt that having it just be `committers-at-large` could be misconstrued as these users can merge whatever they want and not just maintenance related work.